sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -353,6 +353,23 @@ object SQLConf {
       .checkValue(ratio => ratio > 0.0 && ratio <= 1.0, "The ratio value must be in (0.0, 1.0].")
       .createWithDefault(0.04)
 
+  val DYNAMIC_BLOOM_FILTER_JOIN_PRUNING_ENABLED =
+    buildConf("spark.sql.optimizer.dynamicBloomFilterJoinPruning.enabled")
+      .doc("When true, we will generate bloom filter predicate for column when it's used as join " +
+        "key. Note that, dynamic bloom filter join pruning only works if exchange reuse enabled.")
```

Comment:
Use the config property so it becomes `s"only works with ${SQLConf.EXCHANGE_REUSE_ENABLED.key} enabled"`

Reply:
Fixed.